poc-07.py

```diff
@@ -76,6 +76,19 @@ def unmarkImportant(self):
     def unmarkRead(self):
         self.flags['read'] = False
 
+    def mark(self, flag, value):
+        if flag == 'read':
+            if value == True:
+                self.markRead()
+            else:
+                self.unmarkRead()
+
+        elif flag == 'important':
+            if value == True:
+                self.markImportant()
+            else:
+                self.unmarkImportant()
+
```

Comment:
I don't get the point of this method. We already have all the relevant public methods to do this.